AssertionError: must have a panel now!

```
Traceback (most recent call last):
  File "/Users/koenlageveen/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/panel_view.py", line 70, in <lambda>
    sublime.set_timeout_async(lambda: fill_panel(window, update=True))
  File "/Users/koenlageveen/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/panel_view.py", line 275, in fill_panel
    assert panel, "must have a panel now!"
AssertionError: must have a panel now!
```

1. Open a file with highlights
2. Open SublimeLinter settings (e.g. via command palette)
3. Change the style, e.g. specifically for "pylint"
4. Quickly close the settings window
5. Inspect the console

--- a/panel_view.py
+++ b/panel_view.py
@@ -269,10 +269,12 @@ def fill_panel(window, update=False, **panel_filter):
     errors_by_bid = get_window_errors(window, persist.errors, panel_filter)
 
     path_dict, base_dir = create_path_dict(window, errors_by_bid.keys())
-    assert window, "missing window!"
 
     panel = ensure_panel(window)
-    assert panel, "must have a panel now!"
+    # If we're here and the user actually closed the window in the meantime,
+    # we cannot create a panel anymore, and just pass.
+    if not panel:
+        return
 
     settings = panel.settings()
     settings.set("result_base_dir", base_dir)

